### PR TITLE
Pill bottle update

### PR DESCRIPTION
**Pull request by alsoandanswer**
shamelessly stolen from bay, clicking pillbottles in hand will allow you to pour one out into your offhand.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -151,6 +151,26 @@
...
+/obj/item/weapon/storage/pill_bottle/attack_self(mob/living/user)
+	if(user.get_inactive_hand())
+		to_chat(user, "<span class='notice'>You need an empty hand to take something out.</span>")
+		return
+	if(contents.len)
+		var/obj/item/I = contents[1]
+		if(!remove_from_storage(I,user))
+			return
+		if(user.put_in_inactive_hand(I))
+			to_chat(user, "<span class='notice'>You take \the [I] out of \the [src].</span>")
```
Comment (MarinaGryphon) [inline code/game/objects/items/weapons/storage/firstaid.dm]:
```suggestion
			to_chat(user, span("notice","You take \the [I] out of \the [src]."))
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 2]
```diff
@@ -151,6 +151,26 @@
...
+			if(iscarbon(user))
+				var/mob/living/carbon/C = user
+				C.swap_hand()
+		else
+			I.dropInto(loc)
+			to_chat(user, "<span class='notice'>You fumble around with \the [src] and drop \the [I] on the floor.</span>")
```
Comment (MarinaGryphon) [inline code/game/objects/items/weapons/storage/firstaid.dm]:
```suggestion
			to_chat(user, span("notice","You fumble around with \the [src] and drop \the [I] on the floor."))
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 3]
```diff
@@ -151,6 +151,26 @@
...
+	else
+		to_chat(user, "<span class='warning'>\The [src] is empty.</span>")
```
Comment (MarinaGryphon) [inline code/game/objects/items/weapons/storage/firstaid.dm]:
```suggestion
		to_chat(user, span("warning","\The [src] is empty."))
```

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 4]
```diff
@@ -151,6 +151,26 @@
 	drop_sound = 'sound/items/drop/pillbottle.ogg'
 	max_storage_space = 16
 
+/obj/item/weapon/storage/pill_bottle/attack_self(mob/living/user)
+	if(user.get_inactive_hand())
+		to_chat(user, "<span class='notice'>You need an empty hand to take something out.</span>")
```
Comment (MarinaGryphon) [inline code/game/objects/items/weapons/storage/firstaid.dm]:
```suggestion
		to_chat(user, span("notice","You need an empty hand to take something out."))
```

**Pull request merged**